src/matrix.py

```diff
@@ -0,0 +1,14 @@
+class Matrix:
+    def add(self, matrix1, matrix2):
```

Comment:
Is Matrix just doing the addition?
Should it also be used to represent instances of matrices?
In which case the function only needs 1 argument as the other is itself
E.g,
```
matrix1 = Matrix([1, 2][3, 4])
matrix2 = Matrix([5, 6][7, 8])
matrix3 = matrix1.add(matrix2) 
```
A different approach might be to override the __ADD__ function.
```
matrix1 = Matrix([1, 2][3, 4])
matrix2 = Matrix([5, 6][7, 8])
matrix3 = matrix1 + matrix2
```

Reply:
Overriding '+' operator to add two matrices